homeworks/01-twitch/README.md

```diff
@@ -34,6 +34,7 @@ public interface StreamingPlatform {
      * @param title    the title of the stream
      * @param category the {@link Category} of the stream
      * @return the started {@link Stream}
+     * @throws IllegalArgumentException if any of the parameters are null or empty
```

Comment:
Maybe change it to "if any of the parameters are null (or a String is empty)" to avoid ambiguity, what does empty mean for a Category.